modules/networking/public_services_alb.tf

```diff
@@ -15,23 +10,38 @@ resource "aws_alb" "alb_public_services" {
   }
 }
 
-# Listener with empty dummy target group
-resource "aws_alb_target_group" "tgr_dummy_public_services" {
-  name     = "${var.stack_name}-igress-dummy${var.unique_postfix}"
-  port     = local.dummy_port
+resource "aws_autoscaling_attachment" "asg_attachment_public_services" {
+  autoscaling_group_name = var.asg_name_public_services
+  alb_target_group_arn   = aws_alb_target_group.tgr_public_services.arn
+}
+
+resource "aws_alb_target_group" "tgr_public_services" {
+  name     = "${var.stack_name}-ingress-${var.unique_postfix}-${substr(uuid(),0 ,4)}"
+  port     = 9998
   protocol = "HTTP"
   vpc_id   = aws_vpc.vpc_main.id
 
+  health_check {
+    path = "/health"
+    port = 9998
+    timeout = 2
+    interval = 10
+    matcher = "200-299"
```

Comment:
Please fit the match to status code 200 only, since this is the expected return code.